README.md

```diff
@@ -1,2 +1,47 @@
 # capstone-project-yh
+
 Module 1 Microverse Capstone Project
+
+## Built With
+
+- html
+- css
+- javascript
+- linters
+
+## Live Demo
+
+[Click here to see the Portfolio Web](https://yersel500.github.io/capstone-project-yh/)
+
+![image](https://image.prntscr.com/image/ydxJGxH3SNC520UtsK11gw.png)
+
+[Video of the project](https://www.loom.com/share/38c974bd76f948f693195491251ac9de)
```

Comment:
## GOOD JOB SO FAR 🙌
- [ ] [OPTIONAL] Your presentation is awesome and also helped me review your code better 😀. Keep on creating these kinds of great projects 🥇